atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts

```diff
@@ -0,0 +1,131 @@
+import styled from '@emotion/styled'
+import { fluidRange } from 'polished'
+
+import { Avatar, Button } from '@/shared/components'
+import { breakpoints, colors, spacing, typography } from '@/shared/theme'
+import { Link } from '@reach/router'
+
+export const Container = styled.section`
+  position: relative;
+  @media screen and (min-width: ${breakpoints.small}) {
```

Comment:
It looks to me that margins and gradient % need to be in sync based on what "looks good" this is hard to grasp without context, so maybe let's add a comment here describing that.